rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/Transaction.java

```diff
@@ -151,6 +153,14 @@ public boolean save_state(OutputObjectState os, int ot) {
                 os.packBoolean(true);
                 os.packString(status.name());
             }
+
+            if (statusReason == null) {
```

Comment:
Use an enum to model the fixed finite set of reasons for the LRA to be in a given state. Then you will never need to pack a null value.